homeassistant/components/ps4/media_player.py

```diff
@@ -153,6 +155,17 @@ def update(self):
         if status is not None:
             self._retry = 0
             if status.get('status') == 'Ok':
+                # Check if only 1 device in Hass.
+                if len(self.hass.data[PS4_DATA].devices) == 1:
+                    # Enable keep alive feature for PS4 Connection.
+                    # Only 1 device is supported, Since have to use port 997.
+                    self._ps4.keep_alive = True
+                else:
+                    self._ps4.keep_alive = False
+                if self._power_on is True:
```

Comment:
```py
if self._power_on:
```

Reply:
Changed. I thought it was better to be explicit?